components/modal/demo/footer.md

```diff
@@ -1,10 +1,20 @@
 ---
 order: 2
-title: 自定义页脚
+title: 
+  zh-CN: 自定义页脚
+  en-US: Customized footer
 ---
 
+## zh-CN
+
 更复杂的例子，自定义了页脚的按钮，点击提交后进入 loading 状态，完成后关闭。
 
+## en-US
+
+A more complex example, in this example, we define a customized footer button bar,
```

Comment:
A more complex case, as illustrated in this example, we define a customized footer button bar.